addons/account/static/src/components/section_and_note_fields_backend/section_and_note_fields_backend.js

```diff
@@ -99,6 +99,10 @@ export class SectionAndNoteListRenderer extends ListRenderer {
             (editedRecord) => this.focusToName(editedRecord),
             () => [this.editedRecord]
         );
+
+        useSubEnv({
+            shouldCollapse: this.shouldCollapse.bind(this),
+        });
```

Comment:
Can't we move this subEnv in the `SaleOrderLineListRenderer` for now, since it's only useful for the product configurator logic in sale? :thinking: